### PR TITLE
Emotion 11 Snapshots

### PR DESCRIPTION
**Pull request by ryanmccombe**
Snapshots for https://github.com/bbc/simorgh/pull/8742

### CODE REVIEW

File: src/app/Layouts/__snapshots__/defaultPageWrapper.test.jsx.snap [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`defaultPageWrapper should render default page wrapper with children 1`] = `
-.emotion-208 {
+.emotion-0 {
```
Comment (ryanmccombe) [inline src/app/Layouts/__snapshots__/defaultPageWrapper.test.jsx.snap:4]: -1

File: src/app/components/ErrorMain/__snapshots__/index.test.jsx.snap [hunk 2]
```diff
@@ -1559,66 +1541,84 @@ exports[`ErrorMain should correctly render for an error page for urdu 1`] = `
...
         </li>
       </ul>
       <p
-        class="emotion-4 emotion-5"
+        class="emotion-10 emotion-11"
       >
         You can do 
         <a
-          class="emotion-10 emotion-11"
+          class="emotion-18 emotion-19"
```
Comment (ryanmccombe) [inline src/app/components/ErrorMain/__snapshots__/index.test.jsx.snap:1621]: 0

File: src/app/components/Grid/__snapshots__/index.test.jsx.snap [hunk 3]
```diff
@@ -267,113 +267,113 @@ exports[`Grid should render CPSPageGrid with additional props 1`] = `
...
 
 <div>
   <div
-    class="emotion-2 emotion-1"
+    class="emotion-0 emotion-1"
     dir="ltr"
   >
     <div
-      class="emotion-0 emotion-1"
+      class="emotion-2 emotion-1"
```
Comment (ryanmccombe) [inline src/app/components/Grid/__snapshots__/index.test.jsx.snap:376]: 0

File: src/app/containers/AVPlayer/__snapshots__/index.test.jsx.snap [hunk 4]
```diff
@@ -136,15 +136,15 @@ exports[`AVPlayer for On Demand TV should match snapshot for canonical AVPlayer
...
+    class=" emotion-0 emotion-1"
   >
     <div
       class="emotion-2 emotion-3"
     >
       <iframe
         allow="autoplay"
         allowfullscreen=""
-        class="emotion-0 emotion-1"
+        class="emotion-4 emotion-5"
```
Comment (ryanmccombe) [inline src/app/containers/AVPlayer/__snapshots__/index.test.jsx.snap:147]: 0

File: src/app/containers/Ad/Amp/__snapshots__/index.test.jsx.snap [hunk 5]
```diff
@@ -172,70 +196,46 @@ exports[`AMP Ads Snapshots should correctly render an AMP mpu ad 1`] = `
...
       >
         <div
           amp-access="toggles.ads.enabled"
           amp-access-hide="true"
-          class="emotion-2 emotion-3"
+          class="emotion-6 emotion-7"
         >
           <a
-            class="emotion-0 emotion-1"
+            class="emotion-8 emotion-9"
```
Comment (ryanmccombe) [inline src/app/containers/Ad/Amp/__snapshots__/index.test.jsx.snap:238]: 0

File: src/app/containers/CpsFeaturesAnalysis/__snapshots__/index.test.jsx.snap [hunk 6]
```diff
@@ -2,257 +2,260 @@
 
 exports[`CpsRelatedContent should render Story Feature components when given appropriate data 1`] = `
 <DocumentFragment>
-  .emotion-59 {
+  .emotion-0 {
```
Comment (ryanmccombe) [inline src/app/containers/CpsFeaturesAnalysis/__snapshots__/index.test.jsx.snap:5]: -6

File: src/app/containers/CpsOnwardJourney/__snapshots__/index.test.jsx.snap [hunk 7]
```diff
@@ -1,215 +1,215 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`CpsOnwardJourney Creates multiple promos with minimal props 1`] = `
-.emotion-4 {
+.emotion-0 {
```
Comment (ryanmccombe) [inline src/app/containers/CpsOnwardJourney/__snapshots__/index.test.jsx.snap:4]: -11

File: src/app/containers/CpsRecommendations/__snapshots__/index.test.jsx.snap [hunk 8]
```diff
@@ -8,140 +8,140 @@ exports[`CpsRecommendations should not render when there is no recommendations d
 
 exports[`CpsRecommendations should render when cpsRecommendations toggle is enabled and the service supports recommendations for a single item 1`] = `
 @media (max-width:14.9375rem) {
-  .emotion-43 {
+  .emotion-0 {
```
Comment (ryanmccombe) [inline src/app/containers/CpsRecommendations/__snapshots__/index.test.jsx.snap:11]: -3

**Review by jroebu14 — COMMENTED** [review]
I scrutinised all 80 thousand diffs and didn't see anything out of the ordinary 💯

**Review by jroebu14 — APPROVED** [review]

**Review by FK78 — APPROVED** [review]

**Pull request closed without merging**